climpred/bootstrap.py

```diff
@@ -777,7 +890,7 @@ def bootstrap_perfect_model(
         bootstrap=bootstrap,
         pers_sig=pers_sig,
         compute=compute_perfect_model,
-        resample_uninit=bootstrap_uninit_pm_ensemble_from_control,
+        resample_uninit=bootstrap_uninit_pm_ensemble_from_control_cftime,
```

Comment:
Do we need the original `bootstrap_uninit_pm_ensemble_from_control` if this is now the default? Are we expecting that PM is always in `cftime` now as well and can just route it through this function always with the nice stacking hack?

Reply:
Actually this function can go back into a test. 

PM always expects cftime now.

Reply:
So will you move `bootstrap_uninit_pm_ensemble_from_control` to `climpred/tests`? Just checking on this.

Reply:
moved it into a test. also found that I didnt change this in classes...